src/components/chart/column-chart.js

```diff
@@ -15,31 +15,33 @@ class ColumnChart {
         };
     };
 
-    // Set the point padding between each bar to be 3% (an overall gap of 6%)
-    // For charts with fewer than 5 categories, we use a wider point padding of 4% (8% gap between bars)
-    // For cluster charts we use 0 for the point padding and a group padding of 4% (8% gap between bars)
-    updatePointPadding = (config, currentChart, stackedLayout, numberOfExtraLines) => {
+    // Set the point padding between each bar
+    // For charts with fewer than 5 categories, we use a wider point padding of 2%
+    // For cluster charts we use 0 for the point padding and a group padding of 2%
+    updatePointPadding = (config, currentChart, stackedLayout, numberOfExtraLines, isMobile = false) => {
         const numberOfCategories = config.xAxis.categories.length;
         const numberOfSeries = currentChart.series.length - numberOfExtraLines; // Get number of column series
+        // Guidelines
+        const categoryThreshold = isMobile ? 10 : 20;
```

Comment:
The logic to test for `isMobile` isn't working here. It's true the entire time that you call this from the window resize event. Rather than call this on resize and load, I think we might need to refactor things a bit, so that we can set the config initially for both desktop and mobile before we call `Highcharts.chart`. It will take a bit of refactoring though. I'll send some example code.

Reply:
See my demo (which needs some tidying up) of how this could be refactored: https://github.com/ONSdigital/design-system/pull/3557